src/main/webapp/populateData.js

```diff
@@ -329,3 +329,45 @@ function populateGo() {
         }
       });
 }
+
+/**
+ * Populate Plan-mail panel with potential times to read the emails
+ */
+function populatePlanMail() {
+  const planContainer = document.querySelector('#plan');
+  fetch('/plan-mail')
+      .then((response) => {
+        // If response is a 403, user is not authenticated
+        if (response.status === 403) {
+          throw new AuthenticationError();
+        }
+        return response.json();
+      })
+      .then((planMailResponse) => {
+        // Display the potential times to create events
+        const numWordsContainer = document.querySelector('#word-count');
+        numWordsContainer.innerText = planMailResponse.wordCount;
+        const averageReplyContainer = document.querySelector('#average-reply');
+        averageReplyContainer.innerText = planMailResponse.averageReadingSpeed;
+        const timeNeededContainer = document.querySelector('#time-needed');
+        timeNeededContainer.innerText = planMailResponse.minutesToRead;
+        const intervalContainer = document.querySelector('#free-interval');
+        intervalContainer.innerHTML = '';
+        for (const index in planMailResponse.potentialMeetingTimes) {
+          if (typeof index == 'string') {
```

Comment:
nit: I don't think this type check is needed

Reply:
Me neither. But using a `for-in` loop, it is necessary to do it to pass the `lint.js` check

Reply:
That's strange - removing the `if (typeof index == 'string')` shouldn't cause that issue. 

If this has to stay, use `===` then to avoid type coercion

Reply:
Did you figure out what's going on here? I also pointed this out in another PR - it seems strange that the linter requires this check. (Also agree with using strict equality if this is needed.)